modules/ardrivo/src/Wire.cxx

```diff
@@ -0,0 +1,147 @@
+#include "..\..\..\include\ardpolyfills\Wire.h"
+#include "Wire.h"
+#include "Arduino.h"
+#include "BoardDataDef.hxx"
+
+//What sets bus_id?? where does it update?
+//use protocolrequest to track writing to tx?
+void TwoWire::begin()
+{
+    begun = true;
+}
+
+void TwoWire::begin(uint8_t address)
+{
+    slaveAddress = address;
+    begun = true;
+}
+void TwoWire::begin(int address)
+{
+    slaveAddress = address;
+    begun = true;
+}
+
+// requestFrom = REQUEST_SEND
+size_t TwoWire::requestFrom(uint8_t address, size_t size, bool stop)
+{
+ 
+    ProtocolRequest req = ProtocolRequest{ REQUEST_SEND, size };
+    transmissionAddress.value() = address;
```

Comment:
This could unexpectedly throw; use `std::optional<T>::operator=(const T&)` instead